docs/manual/config/index.rst

```diff
@@ -50,7 +50,7 @@ key. The basic operation is:
 * ``mod + <ctrl> + r``: restart qtile with new config
 * ``mod + <group name>``: switch to that group
 * ``mod + <shift> + <group name>``: send a window to that group
-* ``mod + <enter>``: start xterm
+* ``mod + <enter>``: start terminal guessed by `libqtile.utils.guess_terminal`
```

Comment:
I think you want double backticks, not single ones: <code>\`\`libqtile.utils.guess_terminal\`\`</code>

Reply:
Yes I do, fixed